frontend/src/components/EmailIssues.tsx

```
@@ -0,0 +1,36 @@
+
+interface EmailIssuesProps {
+  issues: Issue[];
+}
+
+export function EmailIssues({ issues }: EmailIssuesProps) {
+  if (issues.length === 0) {
+    return null;
+  }
+
+  return (
+    <div className="mt-6" role="status" aria-live="polite">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)